runtime/common/Cargo.toml

```diff
@@ -43,6 +43,8 @@ orml-vesting = {git = "https://github.com/open-web3-stack/open-runtime-module-li
 
 [features]
 default = ["std"]
+frequency = []
+frequency-rococo-testnet = []
```

Comment:
because feature flags are used in runtime package of frequency we need to add these to enable on runtime build which generates wasm